pkg/diagnostics/diagnostic_bundle.go

```diff
@@ -112,6 +112,7 @@ func newDiagnosticBundleFromSpec(af AnalyzerFactory, cf CollectorFactory, spec *
 		WithManagementCluster(spec.Cluster.IsSelfManaged()).
 		WithDefaultAnalyzers().
 		WithDefaultCollectors().
+		WithPackagesCollectors().
```

Comment:
It probably doesnt matter, but just to confirm, is it an issue if we try to collect these and the controller was never installed? i assume troubleshoot will just skip over things that arent there?

Reply:
That's a very good point. Let me try that out.

Reply:
Just verified. Nothing happens when the controller is not installed. The other support items are created.